registrations/serializers.py

```diff
@@ -339,6 +421,63 @@ class Meta:
         fields = ["id", "email", "language"]
 
 
+class RegistrationPriceGroupSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(required=False)
+    price_group = serializers.PrimaryKeyRelatedField(
+        queryset=PriceGroup.objects.all(),
+    )
+    price = serializers.DecimalField(
+        required=False, max_digits=19, decimal_places=2, min_value=0
+    )
+    price_without_vat = serializers.DecimalField(
+        required=False, read_only=True, max_digits=19, decimal_places=2
+    )
+    vat = serializers.DecimalField(
+        required=False, read_only=True, max_digits=19, decimal_places=2
+    )
+    vat_percentage = serializers.DecimalField(
+        required=False,
+        max_digits=4,
+        decimal_places=2,
+        min_value=0,
+        max_value=Decimal("99.99"),
+    )
+
+    def validate(self, data):
```

Comment:
API could also check that unique constraint "unique_registration_price_group" is not violated. At the moment 500 error is returned if user tries to save 2 registration price groups with same price group id. e.g.
```
[
    {
        "id": 6,
        "price": "12.00",
        "price_group": 2,
        "vat_percentage": "14.00"
    },
    {
        "price": 12,
        "price_group": 2,
        "vat_percentage": "0.00"
    }
]
```

Reply:
Added this kind of a validation to `RegistrationSerializer`.